src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -593,7 +593,9 @@ public static PSDataCollection<PSObject> GetPSDataCollection(object inputValue)
         /// <param name="second">Object to compare first to.</param>
```

Comment:
@daxian-dbw, your last commit had 1 failures in `PowerShell-CI-linux`
Validate start of console host.No new assemblies are loaded
```powershell
Expected exactly $null, but got @(@{InputObject=System.Diagnostics.TraceSource.dll; SideIndicator==>}, @{InputObject=System.Runtime.Serialization.Primitives.dll; SideIndicator==>}).
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Startup.Tests.ps1: line 125
125:         $diffs | Should -BeExactly $null
```